### PR TITLE
Added Configure Remote guide.

### PR DESCRIPTION
**Pull request by chhavip**
Added a guide for users to be able to sync their local copy with original remote by adding an upstream.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,31 @@ and self-esteem by navigating the life of their avatar!
...
+## Configure remotes
+When a repository is cloned, it has a default remote called `origin` that points to your fork on GitHub, not the original repository it was forked from. To keep track of the original repository, you should add another remote named `upstream`:
+
+Open terminal or git bash in your local repository and type:
+
+`git remote add upstream https://github.com/systers/powerup-android.git`
+
+Run `git remote -v` to check the status, you should see something like the following:
+
+origin    https://github.com/YOUR_USERNAME/powerup-android.git (fetch)
```
Comment (alma-castant) [inline README.md]: Can you add these four lines inside a code snipet so it's clear that it's not something you write but something you see?

File: README.md [hunk 2]
```diff
@@ -10,6 +10,31 @@ and self-esteem by navigating the life of their avatar!
 1. Clone the repo to your desktop `git clone git@github.com:your_name/powerup-android.git`
 1. Open the project with Android Studio 
 
+## Configure remotes
+When a repository is cloned, it has a default remote called `origin` that points to your fork on GitHub, not the original repository it was forked from. To keep track of the original repository, you should add another remote named `upstream`:
+
+Open terminal or git bash in your local repository and type:
+
+`git remote add upstream https://github.com/systers/powerup-android.git`
```
Comment (alma-castant) [inline README.md]: I would probably make this a list with a point per task you need to accomplish

Comment (chhavip) [inline README.md]: I am not quite clear on what you meant by this, please elaborate

Comment (alma-castant) [inline README.md]: You have some steps here (open terminal, type something, run something else, you should see something, type something else...). Just add `1.` in front of each of them to create a list, so it's easier to read and follow

**Comment by chhavip** [comment]
@alma-castant updates have been made but I dont know how to squash the commits as I have made the changes through online editor.

**Comment by alma-castant** [comment]
Looks good to me :) Can you create a new PR with all of it in a single commit?

**Comment by chhavip** [comment]
@alma-castant , have created a new PR with all changes in one commit. #43

**Pull request closed without merging**